File: CSharp/Palindrome.cs
using System.Linq;

namespace CSharp
{
    // A palindrome is a word that is identical forward and backwards (examples: mom, kayak, racecar).
    // Given a word, create a function that checks whether it is a palindrome.
    // https://edabit.com/challenge/sTTJNhDFDzCDxeiGZ
    public static class Palindrome
    {
        public static bool CheckPalindrome(string str) => str == new string(str.ToCharArray().Reverse().ToArray());
    }
}

File: js/palindrome.js
// A palindrome is a word that is identical forward and backwards (examples: mom, kayak, racecar).
// Given a word, create a function that checks whether it is a palindrome.
// https://edabit.com/challenge/zvFri3sia4iNPTeqC

const checkPalindrome = (str) => str === [...str].reverse().join("");

module.exports = checkPalindrome;
